_config.yml

```diff
@@ -1 +1,56 @@
-theme: jekyll-theme-architect
\ No newline at end of file
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+#
+# If you need help with YAML syntax, here are some quick references for you:
+# https://learn-the-web.algonquindesign.ca/topics/markdown-yaml-cheat-sheet/#yaml
+# https://learnxinyminutes.com/docs/yaml/
+#
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+
+title: The Checker Framework
+email: your-email@example.com
+description: >- # this means to ignore newlines until "baseurl:"
+  The Enforcing, Inferring, and Synthesizing Optional Properties (EISOP) Framework project aims to provide a unified framework for optional properties in Java.
+#baseurl: "opprop-benchmarks.github.io" # the subpath of your site, e.g. /blog
+#baseurl: ""
+url: "" # the base hostname & protocol for your site, e.g. http://example.com
+twitter_username: jekyllrb
+github_username:  jekyll
```

Comment:
Where do these show up?

Reply:
They're default fields that make it easier to add Twitter/GitHub links to the website. If we don't want to use that functionality, I can simply remove them.